src/main/java/com/hillel/dzhurynskyi/homework/hm12/Apple.java

```diff
@@ -0,0 +1,8 @@
+package com.hillel.dzhurynskyi.homework.hm12;
+
+public class Apple extends Fruit {
+    public Apple() {
+        setWeight(1.0F);
```

Comment:
try to avoid magic numbers in the code. use a constant instead:
private static final int APPLE_WEIGHT = 1.0F;